.gitignore

```diff
@@ -24,3 +24,5 @@ bin
 *~
 
 pipelines/
+
+insecure_key
```

Comment:
Why do we need to ignore the insecure_key?

Reply:
Prevent me from uploading because of misoperation.